src/common/userExperience.ts

```diff
@@ -187,3 +191,37 @@ export const experienceTypeToRepositoryMap: Record<UserExperienceType, string> =
     [UserExperienceType.Publication]: 'UserPublicationExperience',
     [UserExperienceType.Course]: 'UserCourseExperience',
   };
+
+// Work Email Verification
+export const completeVerificationForExperienceByUserCompany = async (
+  con: DataSource,
+  { companyId, userId, email: verificationEmail }: UserCompany,
+): Promise<boolean> => {
+  if (!companyId) return false;
+
+  const experience = await queryReadReplica(con, ({ queryRunner }) =>
```

Comment:
read replica could be out of date, since its in a mutation and crucial to get up to date info lets read from master replica